io.sourceforge.RedNotebook.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: io.sourceforge.RedNotebook
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+command: rednotebook
+rename-appdata-file: rednotebook.appdata.xml
+rename-desktop-file: rednotebook.desktop
+rename-icon: rednotebook
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Config access
+  - --filesystem=~/.rednotebook
```

Comment:
This is not a pure config dir. It is rather the equivalent of `~/Notebooks` from **Zim desktop** or `~/Nextcloud` from **QOwnNotes** - the location where notes are saved. Literally no one uses `XDG_CONFIG_DIR` for this.
Moreover, users should expect that after installing the flatpak package, their existing notes will be available.
The main problem with this application is that users have been left with outdated DEB packages. Thanks to the Flatpak/Flathub, they can count on regular updates.

Reply:
> Moreover, users should expect that after installing the flatpak package, their existing notes will be available.

Well, I disagree about it, and it breaks expectations about containing configuration files. It can be noted in project documentation how to do that but it's bad default nonetheless.